v6版本无法在 pnpm hoist:false 下通过 vite 编译

### Reproduction link

[https://github.com/liuweiGL/antd-bug](https://github.com/liuweiGL/antd-bug)

### Steps to reproduce

```sh
git clone https://github.com/liuweiGL/antd-bug.git
pnpm i
pnpm dev
```
### What is expected?

正常启动开发环境

### What is actually happening?

 [ERROR] Could not resolve "@babel/runtime/helpers/interopRequireDefault"

    node_modules/.pnpm/antd@6.0.0_react-dom@19.2.0_react@19.2.0__react@19.2.0/node_modules/antd/lib/locale/zh_CN.js:3:37:
      3 │ ...eDefault = require("@babel/runtime/helpers/interopRequireDefault");
        ╵                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

  You can mark the path "@babel/runtime/helpers/interopRequireDefault" as external to exclude it from the bundle, which will remove this error and leave the unresolved path in the bundle. You can also surround this "require" call with a try/catch block to handle this failure at run-time instead of bundle-time.


    at failureErrorWithLog (C:\Users\wei24\Desktop\antd-demo\node_modules\.pnpm\esbuild@0.25.12\node_modules\esbuild\lib\main.js:1467:15)
    at C:\Users\wei24\Desktop\antd-demo\node_modules\.pnpm\esbuild@0.25.12\node_modules\esbuild\lib\main.js:926:25 
    at C:\Users\wei24\Desktop\antd-demo\node_modules\.pnpm\esbuild@0.25.12\node_modules\esbuild\lib\main.js:1345:9 
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5) {
  errors: [Getter/Setter],
  warnings: [Getter/Setter]
}

| Environment | Info |
| --- | --- |
| antd | 6.0.0 |
| React | 19.2.0 |
| System | windows11 |
| Browser | chrome |

---
https://github.com/ant-design/ant-design/issues/47569

<!-- generated by ant-design-issue-helper. DO NOT REMOVE -->

--- a/package.json
+++ b/package.json
@@ -115,6 +115,7 @@
     "@ant-design/fast-color": "^3.0.0",
     "@ant-design/icons": "^6.1.0",
     "@ant-design/react-slick": "~1.1.2",
+    "@babel/runtime": "^7.28.4",
     "@rc-component/cascader": "~1.8.0",
     "@rc-component/checkbox": "~1.0.0",
     "@rc-component/collapse": "~1.1.1",

--- a/package.json
+++ b/package.json
@@ -115,7 +115,7 @@
     "@ant-design/fast-color": "^3.0.0",
     "@ant-design/icons": "^6.1.0",
     "@ant-design/react-slick": "~1.1.2",
-    "@babel/runtime": "^7.24.7",
+    "@babel/runtime": "^7.28.4",
     "@rc-component/cascader": "~1.8.0",
     "@rc-component/checkbox": "~1.0.0",
     "@rc-component/collapse": "~1.1.1",

--- a/package.json
+++ b/package.json
@@ -115,7 +115,7 @@
     "@ant-design/fast-color": "^3.0.0",
     "@ant-design/icons": "^6.1.0",
     "@ant-design/react-slick": "~1.1.2",
-    "@babel/runtime": "^7.28.4",
+    "@babel/runtime": "^7.24.7",
     "@rc-component/cascader": "~1.8.0",
     "@rc-component/checkbox": "~1.0.0",
     "@rc-component/collapse": "~1.1.1",

